bootstrap.d/dev-libs.yml

```diff
@@ -794,7 +794,34 @@ packages:
       - args: ['make', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
-
+  - name: libretls
```

Comment:
This port lacks meta data. Is it necessary to use this package? We already have OpenSSL.

Reply:
libretls is a libtls reimplementation of libtls for openssl

Reply:
Aside from missing metadata and architecture labels, ports should have one empty line between them. And yes, we need `libretls` or make `telescope` link against `libressl`, which I'm against. Gentoo has `libretls` in their repo's too, so this is fine.